src/connectionconfig/connectionconfig.ts

```diff
@@ -150,7 +150,7 @@ export class ConnectionConfig implements IConnectionConfig {
         // Save the file
         const self = this;
         return new Promise<void>((resolve, reject) => {
-            self._vscodeWrapper.getConfiguration(Constants.extensionName).update(Constants.connectionsArrayName, profiles, true).then(() => {
+            self._vscodeWrapper.getConfiguration(Constants.extensionName).update(Constants.connectionsArrayName, profiles, vscode.ConfigurationTarget.Global).then(() => {
```

Comment:
I meant update this to call _vscodeWrapper.setConfiguration directly

Reply:
I did that and the tests started to fail for some reason so I reverted it